backend/molgenis-emx2-io/src/test/java/org/molgenis/emx2/io/Emx2MembersTest.java

```diff
@@ -0,0 +1,63 @@
+package org.molgenis.emx2.io;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections4.IteratorUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.molgenis.emx2.*;
+import org.molgenis.emx2.io.emx2.Emx2Members;
+import org.molgenis.emx2.io.tablestore.TableStoreForCsvInMemory;
+import org.molgenis.emx2.sql.TestDatabaseFactory;
+
+class Emx2MembersTest {
+
+  private Schema schema;
+  private TableStoreForCsvInMemory store;
+
+  @BeforeEach
+  void setup() {
+    Database database = TestDatabaseFactory.getTestDatabase();
+    schema = database.dropCreateSchema(Emx2MembersTest.class.getSimpleName());
+    store = new TableStoreForCsvInMemory();
+  }
+
+  @Test
+  void shouldInputRoles() {
+    schema.addMember("bofke", "Viewer");
+    Emx2Members.inputRoles(store, schema);
+    List<Member> members = schema.getMembers();
+    assertEquals(members, List.of(new Member("bofke", "Viewer")));
+  }
+
+  @Test
+  void shouldOutputRoles() {
+    schema.addMember("bofke", "Viewer");
```

Comment:
should just have made two tests, one with sufficient level role and one without. That should have given you different outputs.